### ISSUE TITLE
JEI auto fill recipe not working for the Inferium Essences

### ISSUE THREAD

**Issue by Qwetzle** [BlakeBr0/MysticalAgriculture#372]
The essences do not Shift-Fill. I would guess most likely because of the crystals

**Comment by BlakeBr0** [comment]
Yeah, I think it's safe to assume it's because the crystals don't stack. I'm not too sure if there's a way for me to make this work on my end.

**Comment by CKNoah** [comment]
You gotta put the crystal in manually, although maybe the master infusion crystal will let you shift click it.

**Comment by Chromayx** [comment]
any way to make the crystal as a 64 unit item?

**Comment by bramkoekenpan** [comment]
EMI works with this, its sad jei doesnt
